tox.ini

```diff
@@ -9,6 +9,10 @@ deps =
     mozprocess >= 0.22
     moztest >= 0.7
     wptserve >= 1.4.0
+    pytest
+    coverage
+    pytest-cov
 
 commands =
-    {toxinidir}/tests/test.py
+        py.test --cov=mozdownload
+        coverage report
```

Comment:
I would suggest to remove coverage for now given that it still does not work. Also the second line here would result in a duplicated output of the report.